tools/coverity.sh

```diff
@@ -135,8 +140,9 @@ _upload()
 	  --form version=$SHA \
 	  --form description="Travis CI build" \
 	  $UPLOAD_URL)
+	printf "\033[33;1mThe response is\033[0m\n%s\n" "$response"
 	status_code=$(echo "$response" | sed -n '$p')
-	if [ "$status_code" != "201" ]; then
+	if [ "$status_code" != "200" ]; then
```

Comment:
I should probably add a comment explaining why `200` is used here instead of `201` and that is should be changed after the API is fixed. I'll do it tomorrow.